sw/device/lib/base/hardened.h

```diff
@@ -304,4 +304,158 @@ inline void barrier32(uint32_t val) { asm volatile("" ::"r"(val)); }
  */
 inline void barrierw(uintptr_t val) { asm volatile("" ::"r"(val)); }
 
+/**
+ * A constant-time, 32-bit boolean value.
+ *
+ * Values of this type MUST be either all zero bits or all one bits.
+ */
+typedef uint32_t ct_bool32_t;
```

Comment:
Can you add to the comment which value is `true` and which is `false`.

Reply:
Done.